easybuild/easyconfigs/e/EasyQC/EasyQC-9.2-intel-2016b-R-3.3.1.eb

```diff
@@ -0,0 +1,32 @@
+# EasyBuild recipy for EasyQC 
+# Author: Caspar van Leeuwen
+# SURFsara - The Netherlands
+
+easyblock = 'RPackage'
+
+name = 'EasyQC'
+version = '9.2'
+rver = '3.3.1'
+versionsuffix = '-R-%s' % rver
+
+homepage = 'http://www.uni-regensburg.de/medizin/epidemiologie-praeventivmedizin/genetische-epidemiologie/software/'
+description = """EasyQC is an R-package that provides advanced functionality to
+ (1) perform file-level QC of single genome-wide association (GWA) data-sets
+ (2) conduct quality control across several GWA data-sets (meta-level QC)
+ (3) simplify data-handling of large-scale GWA data-sets."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['http://homepages.uni-regensburg.de/~wit59712/easyqc/']
+sources = ['EasyQC_%(version)s.tar.gz']
```

Comment:
@casparvl please include a `SHA256` checksum, `eb --inject-checksums` will do this for you